### PR TITLE
[Enhancement] Added GPU examples

### PR DESCRIPTION
**Pull request by aarnphm**
- Contains GPU examples for Tensorflow, PyTorch, and ONNX
- updated `README` accordingly with future merge links

### CODE REVIEW

File: pytorch/news-classification-gpu/train.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import os
```
Comment (parano) [inline pytorch/news-classification-gpu/train.py:1]: let's also add a link to this script file in the main readme next to the notebook links?  

It's great that you have this script in addition to the notebook, I think it's helpful for many users. We were planning to add this for other gallery projects as well, would be great to give it more exposure to users.

Comment (aarnphm) [reply pytorch/news-classification-gpu/train.py:1]: noted.

File: pytorch/news-classification-gpu/train.py [hunk 2]
```diff
@@ -94,44 +93,4 @@ def evaluate(model, data_loader, criterion):
             loss = criterion(predited_label, label)
             total_acc += (predited_label.argmax(1) == label).sum().item()
             total_count += label.size(0)
-    return total_acc / total_count
-
-
-if __name__ == '__main__':
```
Comment (parano) [inline pytorch/news-classification-gpu/train.py]: Oh I think I misunderstood this part - I thought it was the "load date, train, pack, save BentoBundle" workflow in one script

Comment (aarnphm) [reply pytorch/news-classification-gpu/train.py]: Oh I forgot to remove it.

**Comment by parano** [comment]
Thanks @aarnphm - merging this now, let's discuss adding "scripts based examples" for these example projects

**Comment by aarnphm** [comment]
will do

**Pull request merged**